.gitmodules

```diff
@@ -15,7 +15,7 @@
 	url = https://github.com/arangodb/linenoise-ng
 [submodule "libraries/cmake/source/lzma/src"]
 	path = libraries/cmake/source/lzma/src
-	url = https://git.tukaani.org/xz.git
+	url = https://github.com/tukaani-project/xz
```

Comment:
What is correct here anyhow? After the last xz compromise, I don't know which is considered a better upstream

Reply:
As mentioned in the description, they have blocked cloning from their website, you can see the note on the top: https://git.tukaani.org/

Also Github is their primary repository, as stated in `Tukaani on Github` here: https://tukaani.org/
Github has unblocked their repo since a while and it's now in the hands of the original author.

But in general, we have no choice.

Reply:
It does look that way. Thank you